psga/s_aureus/bactopia_one.nf

```diff
@@ -80,7 +80,6 @@ with open('hello.txt', 'w') as outfile:
     # Run extra bactopia commands
     with open('includes.txt', 'w') as includes_file:
         includes_file.write(sample_id)
-    os.system('bactopia --wf amrfinderplus --bactopia local-single-sample --organism Staphylococcus_aureus --include includes.txt')
```

Comment:
Remove this call as we don't collect any data for it.
For the time being, we use the default antimicrobial resistance tool.